deploy

```diff
@@ -51,11 +69,11 @@ PROXY_SITE_ARG=""
 for SITE in "${SITES[@]}"; do
     echo "Bringing up $SITE"
     docker run -d --name "wodin-$SITE" --network=$NETWORK \
-           -v $PWD/config/$SITE:/wodin/config:ro \
+           -v wodin-config:/wodin/config:ro \
            mrcide/wodin:$APP_BRANCH \
```

Comment:
I suppose at some point you might want to bring up different wodin site containers with different app branches..

Reply:
Yeah, I wondered about this - I've got an idea about how it could be done in the configuration, but definitely would want to wait until we have the deploy tool ready